packages/yum/build-rpm.sh

```diff
@@ -155,6 +155,24 @@ name = MariaDB
 baseurl = http://yum.mariadb.org/10.3/${distribution}${distribution_version}-${mariadb_architecture}
 gpgkey=https://yum.mariadb.org/RPM-GPG-KEY-MariaDB
 gpgcheck=1
+REPO
+        run yum install -y MariaDB-devel
+        if [ ${distribution_version} = 6 ]; then
+          run yum install -y cmake28
+        fi
+        ;;
+      mariadb-10.4-${PACKAGE})
```

Comment:
Can we unify `mariadb-10.X-${PACKAGE}` clauses?

Reply:
I've fixed them for simplicity.